install/kubernetes/cilium/charts/hubble-relay/templates/deployment.yaml

```diff
@@ -41,6 +41,7 @@ spec:
             - "serve"
             - "--peer-service=unix://{{ .Values.global.hubble.socketPath }}"
             - "--listen-address={{ .Values.listenHost }}:{{ .Values.listenPort }}"
+            - "--debug={{ .Values.global.debug.enabled }}"
```

Comment:
nit: we can only add this flag if .Values.global.debug.enabled is true.

Feel free to ignore :)